src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/takemanagement/viewmodel/TakeManagementViewModel.kt

```diff
@@ -204,18 +202,18 @@ class TakeManagementViewModel : ViewModel() {
     }
 
     fun editContent(take: Take) {
-        contextProperty.set(TakeContext.EDIT_TAKES)
-        showPluginActive = true
-        editTake.edit(take)
-                .observeOnFx()
-                .subscribe { result ->
-                    showPluginActive = false
-                    when (result) {
-                        EditTake.Result.SUCCESS -> {
-                        }
-                        EditTake.Result.NO_EDITOR -> snackBarObservable.onNext(messages["noEditor"])
-                    }
-                }
+//        contextProperty.set(TakeContext.EDIT_TAKES)
```

Comment:
why are you commenting this out?

Reply:
@sarabiaj @aunger I'm going to go ahead and just resolve all of the comments about TakeManagementViewModel since the entire file got refactored and removed in a later PR anyway.